ddsrecorder/src/cpp/main.cpp

```diff
@@ -43,6 +43,7 @@
 #include "command_receiver/CommandReceiver.hpp"
 #include "tool/DdsRecorder.hpp"
 
+using namespace eprosima;
```

Comment:
Not a good practice, this can create lots of conflicts between fastrtps and fastdds.